config/locales/models/type_de_champ/fr.yml

```
@@ -5,47 +5,47 @@ fr:
     attributes:
       type_de_champ:
         categorie:
-          standard: 'Standard'
-          choice: 'Choix'
+          structure: 'Structure des champs'
           etat_civil: 'État civil'
-          cadre: 'Cadre'
-          file: 'Fichier'
-          paiement_identification: 'Paiement - Identification'
-          referentiel_externe: 'Référentiel externe'
           localisation: 'Localisation'
+          paiement_identification: 'Paiement et Identification'
+          standard: 'Champ standard'
+          pieces_jointes: 'Pièces jointes'
+          choice: 'Choix'
+          referentiel_externe: 'Référentiel externe'
         type_champs:
-          text: 'Texte'
-          textarea: 'Zone de texte'
-          date: 'Date'
-          datetime: 'Date et Heure'
-          number: 'Nombre'
-          decimal_number: 'Nombre décimal'
-          integer_number: 'Nombre entier'
-          checkbox: 'Case à cocher'
+          header_section: 'Titre de section'
+          repetition: 'Bloc répétable'
+          dossier_link: 'Lien vers un autre dossier'
+          explication: 'Explication'
           civilite: 'Civilité'
-          email: 'Email'
+          email: 'Adresse électronique'
           phone: 'Téléphone'
           address: 'Adresse'
-          yes_no: 'Oui/Non'
-          drop_down_list: 'Choix unique'
-          multiple_drop_down_list: 'Choix multiples'
-          linked_drop_down_list: 'Deux menus déroulants liés'
-          pays: 'Pays'
-          regions: 'Régions'
-          departements: 'Départements'
           communes: 'Communes'
-          header_section: 'Titre de section'
-          explication: 'Explication'
-          engagement: 'Engagement'
-          dossier_link: 'Lien vers un autre dossier'
+          departements: 'Départements'
+          regions: 'Régions'
+          pays: 'Pays'
+          iban: 'Numéro Iban'
```

```suggestion
          iban: 'Numéro IBAN'
```

C'est un acronyme, non ?

---

Oui, comme c'est un acronyme on peut écrire IBAN ou Iban. Là l'idée c'était d'être cohérent avec l'autre acronyme juste dessous : Siret
En revanche un sigle qui n'est pas un acronyme, c'est plutôt tout en capitales, RNA par exemple